wfl/utils/save_calc_results.py

```diff
@@ -1,10 +1,11 @@
 import re
+import warnings
 
 from ase.calculators.singlepoint import SinglePointCalculator
 from ase.calculators.calculator import all_properties, PropertyNotImplementedError
 
 per_atom_properties = ['forces', 'stresses', 'charges', 'magmoms', 'energies']
-per_config_properties = ['energy', 'stress', 'dipole', 'magmom', 'free_energy']
```

Comment:
`free_energy` was not read before (no `atoms.get_free energy`). Now it's saved if it's among the properties that were asked to be calculated.